app.js

```diff
@@ -5,6 +5,7 @@ var logger = require('morgan');
 
 var indexRouter = require('./routes/index');
 var usersRouter = require('./routes/users');
```

Comment:
Maybe you could delete this if you are not planning on using it? Since you have your other users route name spaced, you might want to only keep the name spaced one.

Reply:
Ah, good point.  I was thinking I was going to use it, but the project spec called for a namespaced route, so I never ended up using this one.